Qt.py

```diff
@@ -43,6 +43,26 @@ def _pyqt5():
 
 
 def _pyqt4():
+    # Attempt to set sip API v2 (must be done prior to importing PyQt4)
+    try:
+        import sip
```

Comment:
I'm not sure there's a need to try/except for `sip`. I think `sip` must be available for PyQt4 to work.

```bash
$ sudo mv /usr/lib/python2.7/dist-packages/sip.so /usr/lib/python2.7/dist-packages/_sip.so
$ python
Python 2.7.6 (default, Jun 22 2015, 17:58:13) 
[GCC 4.8.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from PyQt4 import QtGui
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named sip
```